CI fails due to failure to install a Python dependency.

The CI cannot find iree-compiler-snapshot. A possible fix is to remove it from the requirements.txt.

```Run python -m pip install -r ${GITHUB_WORKSPACE}/sandbox/requirements.txt
Looking in links: https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html, https://github.com/google/iree/releases
Collecting numpy
  Downloading numpy-1.22.1-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (16.8 MB)
Collecting pybind11
  Downloading pybind11-2.9.0-py2.py3-none-any.whl (210 kB)
Collecting PyYAML
  Downloading PyYAML-6.0-cp39-cp39-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_12_x86_64.manylinux2010_x86_64.whl (661 kB)
ERROR: Could not find a version that satisfies the requirement iree-compiler-snapshot (from versions: none)
ERROR: No matching distribution found for iree-compiler-snapshot
Error: Process completed with exit code 1.```

--- a/requirements.txt
+++ b/requirements.txt
@@ -8,8 +8,9 @@ pybind11
 PyYAML
 
 # IREE Python API.
-iree-compiler-snapshot
-iree-runtime-snapshot
+# TODO: Enable once available again.
+# iree-compiler-snapshot
+# iree-runtime-snapshot
 torch
 torchvision
 torchaudio

